### DISCUSSION TITLE
我们已经成功打入敌人内部了？！

### DISCUSSION THREAD

**Discussion by Xm798** [Arondight/Adachi-BOT#860] [General]
![image](https://user-images.githubusercontent.com/48074861/178895418-0dac2d86-a350-4105-9a15-fbf56b0bff3d.png)
![image](https://user-images.githubusercontent.com/48074861/178895435-4669fcdb-aae6-4473-bf1e-6308db8bdbfb.jpeg)![image](https://user-images.githubusercontent.com/48074861/178895451-ab303721-8595-4531-98e4-0ccb0089dd7c.jpeg)![image](https://user-images.githubusercontent.com/48074861/178895462-53b77a69-4f9f-4a8b-bec2-2af8c145874f.jpeg)

**Comment by Arondight** [comment]
这都是命运石之门的选择
这都是Message ID: ***@***.***&gt;
